app/src/main/java/com/rkm/tasky/util/image/ImageUtils.kt

```diff
@@ -0,0 +1,61 @@
+package com.rkm.tasky.util.image
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.net.Uri
+import androidx.core.net.toUri
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.IOException
+
+private const val IMAGE_THRESHOLD = 1024L * 1024L
```

Comment:
minor - the name `IMAGE_THRESHOLD` is ambiguous as it does not clarify the units of measurement. It could be referring to `bytes` or `pixels` 🤔 How about naming it: `IMAGE_SIZE_MAX_MB` or something similar?